lib/src/ui/advanced_search_form.dart

```diff
@@ -81,6 +81,14 @@ class _AdvancedSearchFormState extends State<AdvancedSearchForm> {
                   children: [
                     Column(
                       children: [
+                        FlatButton(
+                          child: Text('Reset'),
+                          onPressed: () {
+                            setState(() {
+                              _formResult = SearchParameters();
```

Comment:
So this resets back to the most recently save state variables not a new set of blank variables right?

Reply:
@seamuslee001 actually I would say the later where it sets the blank variables of SearchParameters. On simulator if you select any set of filters and then click Reset button it always brings back to the initial state with all filter options.